src/App.js

```diff
@@ -15,17 +17,23 @@ class App extends React.Component {
   getLoco = async e => {
     e.preventDefault();
     
+    try {
+      let API = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_ACCESS_TOKEN}&q=${this.state.searchQuery}&format=json`;
```

Comment:
- Great work! I left some feedback in Canvas regarding line 21. We are using `let` with all uppercase `API`. In this case, we can just change api to be `apiUrl` or just `url`. Uppercase is typically used with `const` variables 👍